climada/hazard/tc_tracks.py

```diff
@@ -1249,12 +1258,70 @@ def from_netcdf(cls, folder_name):
                                "whole life time.")
                 basin = track.basin
                 del track.attrs['basin']
-                track['basin'] = ("time", np.full(track.time.size, basin))
+                track['basin'] = ("time", np.full(track.time.size, basin, dtype="<U2"))
             data.append(track)
         tr = cls()
         tr.data = data
         return tr
 
+    def write_hdf(self, file_name, complevel=4):
+        """Write TC tracks in HDF5 format.
+
+        Parameters
+        ----------
+        file_name: str or Path
+            Path to a new HDF5 file. If it exists already, the file is overwritten.
+        complevel : int
+            Specifies a compression level (0-9) for the gzip compression of the data.
+            A value of 0 or None disables compression. Default: 4
+        """
+        with h5py.File(file_name, 'w') as store:
+            store.create_dataset('size', (), dtype=int, data=self.size)
+
+            LOGGER.info('Writing %d tracks to %s', self.size, file_name)
```

Comment:
```suggestion
```
Is this really necessary info?

Reply:
By default, this log level is not printed to the console. For power users like me who manually decide to enable this log level, it's interesting to know when certain IO operations start and end and how long they take.

Reply:
But then you should use 'debug'? Info gets printed by default in CLIMADA, at least when using it e.g. in a notebook. I think this is too verbose.

Reply:
No, the default log level is `WARNING`, see here:
https://github.com/CLIMADA-project/climada_python/blob/2111c496a77cd2bad2f603f8e28adcf9c7914124/climada/conf/climada.conf#L56

I think that `debug` should be reserved for developing purposes. So you only activate this level when you are working on the package's code. The `info` level is for users of the package who are interested to know what happens inside of the package.